include/quicr/detail/priority_queue.h

```diff
@@ -112,39 +112,39 @@ namespace quicr {
         /**
          * @brief Get the first object from queue
          *
+         * @param elem[out]          Time queue element storage. Will be updated.
+         *
          * @return TimeQueueElement<DataType> value from time queue
          */
-        TimeQueueElement<DataType> Front()
+        void Front(TimeQueueElement<DataType>& elem)
```

Comment:
any reason to change the API ? <..> Front() is more natural api

Reply:
Priority and time queue are not directly part of the API. They are under detail, which should be considered internal. Our docs should be including them much, it at all.